src/Routes/ImageManagerDetail/StatusLabel.js

```diff
@@ -10,7 +15,8 @@ const StatusLabel = ({ status }) => {
   if (composeStatus.includes(status)) {
     icon = statusIcons[status];
     color = statusColors[status];
-    text = status.charAt(0).toUpperCase() + status.slice(1);
+    text = statusMapper[status];
+    text = text.charAt(0).toUpperCase() + text.slice(1);
```

Comment:
I've opened a PR[1] in frontend-components to add new helper function so we don't have to write this function again and again. However let's keep it as it is now and we can refactor this later on.

[1] https://github.com/RedHatInsights/frontend-components/pull/1197